swap/oracle.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package swap
+
+import "time"
+
+const (
+	// maximum validity of the honey price
+	maxPriceAge       = 1 * time.Hour // TODO: this is currently an arbitrary value - currently irrelevant as prices are fixed
+	defaultHoneyPrice = uint64(1)     // default convertion of honey into output currency - currently ETH
+)
+
+// Currency represents a string identifier for currencies
+type Currency string
```

Comment:
Oh right this is obsolete, good catch.

At some point I was thinking that for multicurrency support we might to actually indicate specifically what currency the `Amount` in the Cheque is.

I dropped this idea for now (even if it may very well be important in the future) so it should be removed.